### PR TITLE
[RLlib; Documentation] Minor fixes "rllib in 60s" and per-feature sigils.

### PR DESCRIPTION
**Pull request by sven1977**
Minor fixes "rllib in 60s" and per-feature sigils.
- Add "we're hiring" to all RLlib pages (using shared rst block).
- (tentative) sigils for the different features highlighted (plus google docs link to edit/comment).
- Add inference examples link at the bottom of RLlib-in-60s code example
- Add more links to examples in highlighted features section

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [x] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [x] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [x] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: doc/source/rllib-training.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
+.. admonition:: We're hiring!
```
Comment (gjoliver) [inline doc/source/rllib-training.rst:1]: huh, rst doesn't support something like include another template?

File: doc/source/rllib/index.rst [hunk 2]
```diff
@@ -66,103 +66,115 @@ for a few training iterations, then perform a single evaluation loop
...
-..            :width: 120
+    .. container::
 
-.. container::
+        **External simulators**: Don't have your simulation running as a gym.Env in python?
+        No problem! RLlib supports an external environment API and comes with a pluggable,
+        off-the-shelve client/server setup that allows you to run 100s of independent
+        simulators on the "outside" (e.g. a Windows cloud) connecting to a central RLlib
+        Policy-Server that learns and serves actions. Alternatively, actions can be computed
+        on the client side to save on network traffic.
```
Comment (gjoliver) [inline doc/source/rllib/index.rst:158]: is there an example for this actually? or code pointer?

File: doc/source/rllib/index.rst [hunk 3]
```diff
@@ -66,103 +66,115 @@ for a few training iterations, then perform a single evaluation loop
...
+        .. image:: ../images/rllib/sigils/rllib-sigil-offline-rl.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/offline_rl.py
+
+    .. container::
+
+        **Offline RL and imitation learning/behavior cloning**: You don't have a simulator
+        for your particular problem, but tons of historic data recorded by a legacy (maybe
+        non-RL/ML) system? This branch of reinforcement learning is for you!
+        RLlib's comes with several offline RL algorithms (*CQL*, *MARWIL*, and *DQfD*),
```
Comment (gjoliver) [inline doc/source/rllib/index.rst:174]: shit (note to myself), I really need to get to these algorithms :) 💪

**Review by gjoliver — COMMENTED** [review]
nice change, just a couple of quetsions.

File: doc/source/rllib-training.rst (hunk 1 above)
Comment (sven1977) [reply doc/source/rllib-training.rst:1]: Yeah, was thinking about this, too :D 
Not a sphinx expert, but I think you can define shared blocks and then use those instead. Will fix.

File: doc/source/rllib/index.rst (hunk 2 above)
Comment (sven1977) [reply doc/source/rllib/index.rst:158]: Yeah, I linked the directory with the client/server examples (Unity3D and CartPole) in the sigil.
I can add a link to the text as well.

**Comment by worldveil** [comment]
lgtm, except would be nice to make the "hiring" block a snippet so can change in one place

**Comment by worldveil** [comment]
WOW love the intro page so much now

File: doc/source/rllib/index.rst [hunk 4]
```diff
@@ -66,103 +66,115 @@ for a few training iterations, then perform a single evaluation loop
...
+        .. image:: ../images/rllib/sigils/rllib-sigil-vector-envs.svg
+            :width: 100
+            :target: https://github.com/ray-project/ray/blob/master/rllib/examples/env_rendering_and_recording.py
 
-.. .. container:: clear-both
+    .. container::
 
-..    .. container:: buttons-float-left
+        **Vectorized (batched) and remote (parallel) environments**: RLlib auto-vectorizes
+        your ``gym.Env``s via the ``num_envs_per_worker`` config. Environment workers can
```
Comment (worldveil) [inline doc/source/rllib/index.rst:115]: This doesn't render super well -- do we need to escape the `s` somehow?
Screen Shot 2021-11-12 at 12 16 16 AM

Comment (sven1977) [reply doc/source/rllib/index.rst]: fixed

File: doc/source/rllib/index.rst (hunk 3 above)
Comment (sven1977) [reply doc/source/rllib/index.rst]: :D

File: doc/source/rllib-training.rst (hunk 1 above)
Comment (sven1977) [reply doc/source/rllib-training.rst]: fixed

File: doc/source/rllib/index.rst (hunk 2 above)
Comment (sven1977) [reply doc/source/rllib/index.rst]: done. Pointed directly to the "client" and "server" separate example scripts in the test. The sigil links into the entire "examples/serving" directory.

**Comment by worldveil** [comment]
lgtm!

**Review by gjoliver — APPROVED** [review]
Nice man!!! :)

**Review by worldveil — APPROVED** [review]
LGTM

**Pull request merged**